### DISCUSSION TITLE
Github Pages to host a mobile app privacy policy

### DISCUSSION THREAD

**Discussion by DktPhl2019** [community/community#57365] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello!

I want to use github pages to host a single page privacy policy for my mobile app.

Technically, this is not an online business. 

I would like to hear your thoughts.

Thank You

**Comment by ImaniAN** [comment]
ideally, this should work and I don't think it breaks any terms of service.

lemme know if you need any help... I would be willing to contribute
